pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -165,14 +172,14 @@ let
       # "com_github_googleapis_googleapis"
       # "com_github_googlecloudplatform_google_cloud_cpp"
       "com_github_grpc_grpc"
-      "com_google_protobuf"
+      # Fails with the error: /build/output/external/com_google_protobuf/BUILD.bazel:50:8: in cmd attribute of genrule rule @com_google_protobuf//:link_proto_files: $(PROTOBUF_INCLUDE_PATH) not defined
+      # "com_google_protobuf"
```

Comment:
I wasn't able to get this working with protobuf in nixpkgs, even after trying several variations on the method in the [tensorflow derivation](https://github.com/NixOS/nixpkgs/blob/9da9722e432e40d877e07cdee44e896ae426f874/pkgs/development/python-modules/tensorflow/default.nix#L286-L289).

Reply:
That's a bummer, thanks for taking a good stab at it!

Reply:
The tensorflow derivation runs configure: https://github.com/NixOS/nixpkgs/blob/9da9722e432e40d877e07cdee44e896ae426f874/pkgs/development/python-modules/tensorflow/default.nix#L345, which picks up `PROTOBUF_INCLUDE_PATH` like this: https://sourcegraph.com/github.com/tensorflow/tensorflow@b9c969ef42b3a1172af3a79f3497aee74f3658c0/-/blob/configure.py?L1016-1018, jaxlib doesn't have the equivalent configuration logic, but we can specify `--define=PROTOBUF_INCLUDE_PATH="${protobuf-core}/include"` in the bazel flags explicitly.

Reply:
How `--define` plays into this is a bit poorly documented because its used is discouraged, having been replaced by build settings and toolchains, but make variables in general are documented here: https://bazel.build/reference/be/make-variables, and build variables assigned with `--define` is made available as a make variable.

Reply:
Thanks for the tips @uri-canva ! I'm not sure what changed, but after rebasing I was able to revert this diff (i.e. go back to using nixpkgs protobuf) without the above error.

Reply:
> Thanks for the tips @uri-canva ! I'm not sure what changed, but after rebasing I was able to revert this diff

Hmm, actually I must've been mistaken -- I started seeing the `PROTOBUF_INCLUDE_PATH` errors in the ofborg build even though I could've sworn I got past that locally.

Now I've explicitly set `PROTOBUF_INCLUDE_PATH` in the bazel flags with the method @uri-canva suggested, and that seems to have fixed it.